install-deps.sh

```diff
@@ -551,6 +554,11 @@ if $for_make_check; then
     done
     rm -rf $top_srcdir/install-deps-python3
     rm -rf $XDG_CACHE_HOME
+    if [ -z $ORIGINAL_XDG_CACHE_HOME ]; then
+        unset XDG_CACHE_HOME
+    else
+        export XDG_CACHE_HOME=$ORIGINAL_XDG_CACHE_HOME
+    fi
```

Comment:
```suggestion
    if [ -n "$ORIGINAL_XDG_CACHE_HOME" ]; then
        XDG_CACHE_HOME=$ORIGINAL_XDG_CACHE_HOME
    else
        unset XDG_CACHE_HOME
    fi
```

couple changes:

1. quote variables when check for empty. the same applies to the other change in this commit.
2. no need to `export` a variable if it is already exported
3. favor positive conditions over negative ones

Reply:
changed, please review.